docs/quickstart.mdx

```diff
@@ -28,35 +28,53 @@ You'll see a confirmation:
 ✓ Logged in as User
 ```
 
-## Create Your Project
+## Getting started with Suga
     
-Create a new project from a template:
+Suga can be initialized into an existing project, a new folder, or you can use the new command to scaffold a starter template.
 
-```bash title="Create New Project" icon="sparkles"
-suga new my-first-app
-```
+<Tabs>
+  <Tab title="Initialize your Project">
+    Initialize Suga in your project directory:
 
-Select any template you prefer:
+    ```bash title="Initialize in Existing Project" icon="folder-plus"
+    cd your-existing-app
+    suga init
+    ```
 
-<Info>
-  You're welcome to use frameworks or tools beyond what's listed in the Templates, consider them a starting point or example.
-</Info>
+    This will create a `suga.yaml` file that describes your application's infrastructure requirements.
+  </Tab>
```

Comment:
```suggestion
  </Tab>
  <Tab title="Start with an Empty Project">
    Start a new project from scratch with Suga:

    ```bash title="Start with a Blank Project" icon="folder-plus"
    # Create a new project folder and navigate to it
    mkdir your-project && cd your-project
    
    # Initialize Suga
    suga init
    ```

    This will create a `suga.yaml` file that describes your application's infrastructure requirements.
  </Tab>
```